pvlib/irradiance.py

```diff
@@ -2048,3 +2048,70 @@ def _get_dirint_coeffs():
         [0.743440, 0.592190, 0.603060, 0.316930, 0.794390]]
 
     return coeffs[1:, 1:, :, :]
+
+
+def dni(ghi, dhi, zenith, clearsky_dni=None, clearsky_tolerance=1,
```

Comment:
Would we be better off with a more descriptive name? ``dni_from_components``? I'm not sure.